reporter-config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "reporterEnabled" : "cypress-junit-reporter, cypress-mochawesome-reporter",
+    "cypressJunitReporterReporterOptions" : {"mochaFile": "e2e/reports/xml/e2e-test-output-[hash].xml"},
```

Comment:
We can not define only directory here as it is a file (mochaFile). Also we have to use `[hash]` here because we are running 2 test suites, without this first xml file will be overwritten by second xml file.